src/parsec_client/operations/psa_asymmetric_decrypt.md

```diff
@@ -0,0 +1,40 @@
+# PsaAsymmetricDecrypt
+
+Decrypt a short message with a private key. Opcode: 9 (`0x0009`)
+
+## Parameters
+
+| Name       | Type                                                                      | Description                                                                |
+|------------|---------------------------------------------------------------------------|----------------------------------------------------------------------------|
+| `key_name` | String                                                                    | Name of the key to use for the operation                                   |
+| `alg`      | [`AsymmetricEncryption`](psa_algorithm.md#asymmetricencryption-algorithm) | An asymmetric encryption algorithm that is compatible with the type of key |
+| `message`  | Vector of bytes                                                           | Short message to decrypt                                                   |
```

Comment:
For decrypt we have `message` in the parameters and `decrypted_message` in the results and for encrypt it is `message` and `encrypted_message`. I propose to go for fully descriptive name and use `decrypted_message` and `encrypted_message` only OR use `message` only and let the consumer be smart enough to see that depending on the operation and the parameters/results context what is the type of the message. I think the first option is more friendly 👍

Reply:
How about `ciphertext` and `plaintext`?

Reply:
Although since this has already been resolved, it's probably ok as it is!

Reply:
I've changed to Ionut's suggestion to keep it the same as `parsec-operations`.